src/phasorpy/phasor.py

```diff
@@ -2998,6 +3000,168 @@ def phasor_filter_median(
     return mean, real, imag
 
 
+def phasor_filter_wavelet(
+    mean: ArrayLike,
+    real: ArrayLike,
+    imag: ArrayLike,
+    /,
+    *,
+    sigma: float = 2.0,
+    levels: int = 1,
+    harmonic: int | Sequence[int] | Literal['all'] | str | None = None,
+    skip_axis: int | Sequence[int] | None = None,
+    normalize: bool = False,
+    samples: int = 1,
+    nan_safe: bool = True,
+) -> tuple[NDArray[Any], NDArray[Any], NDArray[Any]]:
+    """Return wavelet-filtered phasor coordinates.
+
```

Comment:
Add a strong note that the function should be used with calibrated but otherwise unprocessed (never filtered or thresholded) FLIM data only.